src/components/ProjectField/ProjectDataDisplay.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { isValidElement, useMemo } from 'react';
+
+import { Box, Grid, Typography, useTheme } from '@mui/material';
+import { Icon, Tooltip } from '@terraware/web-components';
+
+type DataValue = string | number | null | undefined | JSX.Element;
+
+const DataValueElement = ({ value, units }: { value: DataValue; units?: DataValue }) => {
+  if (isValidElement(value)) {
+    return value;
+  }
+
+  const hasValue = useMemo(() => !([undefined, null] as DataValue[]).includes(value), [value]);
+
+  return (
+    <Typography
+      sx={{
+        fontSize: '16px',
+        lineHeight: '32px',
+        fontWeight: 600,
+        overflow: 'hidden',
+        textOverflow: 'ellipsis',
+      }}
+    >
+      {hasValue ? value : 'N/A'} {hasValue && units}
```

Comment:
Almost left a comment saying 'N/A' needs to be localized before remembering it's the same across English, Spanish, and French.